scripts/form-validation.js

```diff
@@ -0,0 +1,11 @@
+const email = document.getElementById('email');
+const form = document.getElementById('form');
+const errorText = document.getElementById('errorTxt');
+const validator = /^([a-z\d-]+)@([a-z\d-]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/;
+
+form.addEventListener('submit', (e) => {
+  if (!(validator.test(email.value))) {
+    errorText.style.display = 'block';
+    e.preventDefault();
+  }
+});
```

Comment:
- [OPTIONAL] Wonder job for utilizing regular expressions to validate your form email input. I would recommend you to consider make the validation more robust to accommodate format like this too :point_right: `qoosim.alaje@gmail.com`. At this moment this format of email is not accepted. Take a look
![validation](https://user-images.githubusercontent.com/26597432/216602588-61a9b6b0-e9ac-4888-a139-426a092231b0.png)